Change data structures to database

allebd opened the pull request:
#### What does this PR do?
Changes data structures to database
#### Description of Task to be completed?
- Changed all data structure implementation to database implementation
- Configure travis for database implementation
#### How should this be manually tested?
NA
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
#165485117
#### Screenshots (if appropriate)
![screenshot of invalid email](https://user-images.githubusercontent.com/7049701/56655090-baad8f00-6689-11e9-8d5f-b1f72e146d52.PNG)
Screenshot of invalid email on signup

![screenshot of signup](https://user-images.githubusercontent.com/7049701/56655106-c39e6080-6689-11e9-9888-5e6706774f9f.PNG)
Screenshot of successful signup

![screenshot of signin](https://user-images.githubusercontent.com/7049701/56655119-cac56e80-6689-11e9-976f-807ac1507178.PNG)
Screenshot of successful sign in

#### Questions:
NA

allebd committed:
chore(database): data structure to database
Changed all data structure implementation to database implementation
Configure travis for database implementation
[Finishes #165485117]

uchemukolo reviewed (commented):
Great job on your implementation so far. I noticed some of the queries were repeated. It is better to create all your queries once in a separate file and import them when you need, making it reusable. 
E.g  `const query = 'SELECT * FROM accounts WHERE accountnumber = $1'` was used twice in the same file.

uchemukolo commented:
You can always add a screenshot of postman results to the PR description

allebd committed:
bug(queries): fix duplicate queries
Fixed all duplicate queries to be reuseable
Reconfigured travis
[Finishes #165560830]
